internal/pwru/ksym.go

```diff
@@ -57,8 +57,12 @@ func ParseKallsyms(funcs Funcs, all bool) (Addr2Name, BpfProgName2Addr, error) {
 
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		line := strings.Split(scanner.Text(), " ")
-		name, isBpfProg := extractBpfProgName(line[2])
+		line := strings.Fields(scanner.Text())
+		isBpfProg := len(line) > 3 && line[3] == "[bpf]"
+		name := line[2]
+		if isBpfProg && len(name) > 26 {
+			name = name[26:]
+		}
```

Comment:
This breaks --filter-track-bpf-helpers because pwru relies on  `[bpf]` suffix to collect bpf functions.